terragrunt/concourse-wg-ci/config.yaml

```
@@ -17,7 +17,7 @@ dns_domain: ci.cloudfoundry.org
 gke_name: wg-ci
 
 # Concourse teams
-concourse_github_mainTeam: "cloudfoundry:wg-app-runtime-interfaces-autoscaler-approvers"
+concourse_github_mainTeam: "cloudfoundry:wg-app-runtime-interfaces-autoscaler-approvers,cloudfoundry:wg-app-runtime-interfaces-capi-approvers"
```

Unfortunately, this `,` needs to be escaped (with an escaped `\`) (see also https://github.com/cloudfoundry/app-runtime-interfaces-infrastructure/blob/main/terragrunt/concourse-wg-ci-test/config.yaml#L20 )

```suggestion
concourse_github_mainTeam: "cloudfoundry:wg-app-runtime-interfaces-autoscaler-approvers\\,cloudfoundry:wg-app-runtime-interfaces-capi-approvers"
```

---

thanks @silvestre  for pointing out. Changes are done as per sugguestions.